pga2d/objects.js

```diff
@@ -47,30 +86,44 @@ export class Point {
     }
   }
 
-  euclidean_norm() {
+  euclidean_norm_sqr() {
     // the xo and yo parts square to 0, so we only have xy * yx
     const { xy } = this.bivec;
     return xy * xy;
   }
 
-  euclidean_mag() {
-    return Math.sqrt(this.euclidean_norm());
+  euclidean_norm() {
+    return Math.sqrt(this.euclidean_norm_sqr());
   }
 
-  ideal_norm() {
+  ideal_norm_sqr() {
```

Comment:
I'm noticing at least one place (`Coral/TileEditor/interactive.js` where I didn't account for this renaming. Double check where I use these things.